pytest/tests/stress/stress.py

```diff
@@ -170,16 +170,29 @@ def monkey_node_restart(stopped, error, nodes, nonces):
 
 @stress_process
 def monkey_local_network(stopped, error, nodes, nonces):
+    last_height = 0
+    last_time_height_updated = time.time()
+
     while stopped.value == 0:
+        _, cur_height = get_recent_hash(nodes[-1], 30)
+        if cur_height == last_height and time.time() - last_time_height_updated > 10:
+            time.sleep(25)
+        else:
+            last_height = cur_height
+            last_time_height_updated = time.time()
+            time.sleep(5)
```

Comment:
This is also different from the original PR, in which it was a constant 9s sleep.
If the cluster got split into two halves with 1 height difference, it takes 8-9 seconds to even detect it, and then some time to recover, so sleeping longer when the cluster stalls.